config/overlays/odh/set-resources-manager-patch.yaml

```diff
@@ -11,4 +11,4 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 500Mi
+            memory: 5Gi
```

Comment:
post mortem/maybe next time: would be nice to have a comment there with the tracking issue directly in the source since it's just a temporary workaround :slightly_smiling_face:

Reply:
Yes, makes sense.
I'll try to not forget next time.